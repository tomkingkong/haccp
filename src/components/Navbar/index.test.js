import React from 'react';
import { shallow } from 'enzyme';

import { Navbar } from '.';

describe('Navbar Component', () => {
  it('should match snapshot', () => {
    const wrapper = shallow(<Navbar />);
    expect(wrapper).toMatchSnapshot();
  });
});