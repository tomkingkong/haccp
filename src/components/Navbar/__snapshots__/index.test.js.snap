// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Navbar Component should match snapshot 1`] = `
<div
  className="Navbar"
>
  <NavLink
    activeClassName="active"
    aria-current="page"
    to="/receiving"
  >
    Receiving
  </NavLink>
  <NavLink
    activeClassName="active"
    aria-current="page"
    to="/receiving"
  >
    Storage
  </NavLink>
  <NavLink
    activeClassName="active"
    aria-current="page"
    to="/receiving"
  >
    Processing
  </NavLink>
  <NavLink
    activeClassName="active"
    aria-current="page"
    to="/receiving"
  >
    Packaging
  </NavLink>
  <NavLink
    activeClassName="active"
    aria-current="page"
    to="/receiving"
  >
    Summary
  </NavLink>
</div>
`;
